Formula/traildb.rb

```diff
@@ -0,0 +1,21 @@
+
+class Traildb < Formula
+  desc "Blazingly-fast database for log-structured data"
+  homepage "http://traildb.io"
+  url "https://github.com/traildb/traildb/archive/0.5.tar.gz"
+  sha256 "4d1b61cc7068ec3313fe6322fc366a996c9d357dd3edf667dd33f0ab2c103271"
+
+  depends_on "libarchive"
+  depends_on "homebrew/boneyard/judy"
```

Comment:
You could install it as a `resource` in this formula, that's probably the best fit I think. Will let other @Homebrew/maintainers express views here too.